wpimath/src/main/native/include/units/base.h

```diff
@@ -462,46 +462,57 @@ namespace units
 	//	DOXYGEN
 	//----------------------------------
 
+	/**
+	 * @defgroup 	Units Unit API
+	*/
+
 	/**
 	 * @defgroup	UnitContainers Unit Containers
+	 * @ingroup		Units
```

Comment:
Alignment of "Units" seems a little strange versus surrounding text?

Reply:
agh, vscode switched to tabs for some reason.

Reply:
oh...this file has mixed tabs and spaces. lovely.